sap-extensions/extensions.json

```diff
@@ -45,12 +45,12 @@
         "x-sap-dpp-is-potentially-personal": {
           "type": "boolean",
           "default": true,
-          "description": "Property contains potentially personal data. Properties annotated with x-sap-dpp-field-semantics need not be additionally annotated with this extension."
+          "description": "Property contains potentially personal data. Properties annotated with x-sap-dpp-field-semantics need not be additionally annotated with this extension. Personal data describes any information which is related to an identified or identifiable natural person (data subject). An identifiable person is one who can be identified, directly or indirectly, in particular by a reference to an identifier such as a name, an identification number, location data, an online identifier, or to one or more factors specific to the physical, physiological, genetic, mental, economic, cultural, or social identity of that natural person."
```

Comment:
Please generate SAP schemas ! run `npm run generate` and commit those files to this PR. Thx.

Reply:
@pavelkornev How about automating that step with a GitHub action?

Reply:
Yeah, i think we should generate markdown files in general and have just one place where we define things, not 2.